(Update) Add featureFlag note for RBAC (#7233)
harness/developer-hub: docs/platform/pipelines/pipeline-settings.md

By default, the pipeline execution audit events such as Pipeline Start, Pipeline End, Stage Start, and Stage End are not displayed in the [Audit Trail](/docs/platform/governance/audit-trail/#step-view-an-audit-trail). Enable this setting to display these events.

### Run RBAC Validation before executing Inline Pipelines

> :::Note

Reviewer: ```suggestion
:::note
```

> Currently, this feature is behind the feature flags CDS_PIPELINE_ABORT_RBAC_PERMISSION_MIGRATION and CDS_PIPELINE_ABORT_RBAC_PERMISSION. Contact Harness Support to enable the feature.

Reviewer: ```suggestion
Currently, this feature is behind the feature flags `CDS_PIPELINE_ABORT_RBAC_PERMISSION_MIGRATION` and `CDS_PIPELINE_ABORT_RBAC_PERMISSION`. Contact Harness Support to enable the feature.
```

:::

By default, before running any pipeline, Harness checks whether the user has access to all of the environments and other resources that the pipeline accesses. This check is run only for inline pipelines, not those stored in Git or other repositories. 

Turn off this setting if the check isn't required. Turning this off can cause pipelines to fail partway through execution (since later stages could have permission issues that haven't been checked beforehand).

> The setting can be found on the Account level Default settings.

Reviewer: ```suggestion
The **Run RBAC Validation before executing Inline Pipelines** setting is available in the account-level default settings.
```

> The **Run RBAC Validation before executing Inline Pipelines** setting is available in the account-level default settings.

Reviewer: ```suggestion
The **Run RBAC Validation before executing Inline Pipelines** setting is available in the account-level default settings.

```
